stubs/pyformance/__init__.pyi

```diff
@@ -5,6 +5,9 @@ from pyformance.meters import Gauge
 
 
 class MetricsRegistry:
+
+    _gauges: dict[str, Gauge]
+
```

Comment:
What does this change bring ? I'm not familiar with `.pyi` files

Reply:
it is typing information: pyformance has no mypy type annotation, in this file, we add them.
So I made this field known to mypy by adding it here.